example.ini

```diff
@@ -27,6 +27,11 @@ web.mac_secret = YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXowMTIzNDU2Nzg5
 ; 55 seconds to maintain a connection. note: this will be jittered a bit.
 web.ping_interval = 45
 
+; b64-encoded auth token for admin-only service APIs
+web.admin_auth = aHVudGVyMg==
```

Comment:
Will this be public facing?  Maybe we should pick a longer one.

Edit: Very funny.